elfpy/utils/apeworx_integrations.py

```diff
@@ -19,12 +20,12 @@
 
 import elfpy
 from elfpy import types
-from elfpy.agents.wallet import Long, Short, Wallet
```

Comment:
Matt has some thing set up to alphabetically sort imports whenever he saves the file. So if this was intentional, just letting you know he will undo it if he ever modifies this file. I tend to just leave it alphabetical.